.github/actions/make_init/action.yml

```diff
@@ -26,7 +26,10 @@ runs:
         cache: 'yarn'
         cache-dependency-path: '**/yarn.lock'
     - name: Initialize React
-      run: make react-init
+      run: |
+        # Create the cache directory if it does not exist.
+        mkdir -p $(yarn cache dir)
```

Comment:
I don't understand why my contribution doesn't work without it even though I didn't change anything in cache yarn, but now it's needed.